planning/behavior_path_avoidance_module/src/utils.cpp

```diff
@@ -250,16 +250,37 @@ void pushUniqueVector(T & base_vector, const T & additional_vector)
   base_vector.insert(base_vector.end(), additional_vector.begin(), additional_vector.end());
 }
 
+bool isAvoidShift(const double start_shift_length, const double end_shift_length)
+{
+  constexpr double THRESHOLD = 0.1;
```

Comment:
```
bool isAvoidShift(const double start_shift_length, const double end_shift_length, const double THRESHOLD = 0.1)
{
  return std::abs(start_shift_length) < THRESHOLD && std::abs(end_shift_length) > THRESHOLD;
}
```

Personally, I would wonder if it is helpful to make threshold a "default" argument for these functions?

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/6300/commits/e677db4309abc6d15f879b0fded9cdf96247ef81.